src/components/annotate/modal/ValueField.tsx

```diff
@@ -0,0 +1,74 @@
+/* Import Dependencies */
+import { Field } from 'formik';
+
+/* Import Types */
+import { Dict } from 'global/Types';
+
+/* Import Sources */
+import AnnotationFilterLayer from 'sources/annotationFilterLayer.json';
+import DigitalMediaFilterLayer from 'sources/digitalMediaFilterLayer.json';
+
+
+/* Props Typing */
+interface Props {
+    name: string,
+    value: string | Date,
+    property: string,
+    targetType: string
+};
+
+
+const ValueField = (props: Props) => {
+    const { name, property, targetType } = props;
+
+    /* Base variables */
+    const annotationFilterLayer: Dict = AnnotationFilterLayer;
+    const digitalMediaFilterLayer: Dict = DigitalMediaFilterLayer;
+    let filteredAnnotation;
+
+    if (targetType === 'digital_specimen') {
+        filteredAnnotation = annotationFilterLayer[property];
+    } else if (targetType === 'digital_media') {
+        filteredAnnotation = digitalMediaFilterLayer[property];
+    }
+
+    /* Function for tracking start date if property is time related */
+    
+    /* Future Development: implement date field
```

Comment:
Ah a nicely commented out piece. Is this still relevant or can it be deleted?

Reply:
I think this is still relevant because there are more types than just string and dates, think of select fields and booleans, point for discussion I guess.

https://naturalis.atlassian.net/browse/DD-257?atlOrigin=eyJpIjoiOGZmMDNmYTUzMWVlNDgxYWI0ZGEyMTM5ZDQ5NDJjMmEiLCJwIjoiaiJ9